MIGRATION.md

```diff
@@ -99,9 +99,9 @@ func Get() (*Config, error) {
  }
 
  if cfg.Debug {
-  cfg.PatternLibraryAssetsPath = "http://localhost:9000/dist"
+  cfg.PatternLibraryAssetsPath = "http://localhost:9001/dist/assets"
```

Comment:
This port conflicts with the [minio](https://github.com/ONSdigital/dp-compose/blob/e845cb15856bfc0334afbb2b6b04ee0547672780/cantabular-import/deps.yml#L42) service in dp-compose/cantabular-import. I have addressed this in my [active PR](https://github.com/ONSdigital/dp-design-system/pull/39). I suggest you update the local port